pytest_order/item.py

```diff
@@ -138,19 +145,21 @@ def handle_relative_marks(
     def group_order(self) -> Optional[int]:
         if self.start_items:
             return self.start_items[0][0]
-        if self.end_items:
+        elif self.end_items:
```

Comment:
Any reason for this?

Reply:
Branch coverage of two if statements and an else requires 4 paths, vs 3 paths for if-elif-else. This technically doesn't matter, as there are returns under every statement, but still.

Reply:
It's ok for me, I'm just to avoid unneeded else statements (I guess that comes from C++/C#, I'm not sure if the generated Python byte code is different in this case).

Reply:
Because of the return statements being there, runtime is going to look exactly the same, which is why the tests pass with this. Also, `elif` is not exactly an `else` in this case, an `elif` expression is going to be checked only if the first one evaluates to `False`, which fits the current usage more than two separate if statements IMO.

Reply:
Yes, I understand that this does not change the logic, was just interested to see if the generated byte code is the same - though I can easily test this... Anyway, that was just out of interest, I'm fine with it.